portfolio/src/main/webapp/script.js

```diff
@@ -27,10 +27,38 @@ function addRandomQuote() {
       Flying blindly into the abyss, \
       believing therein lie the answers to the mysteries of the universe.'];
 
-  // Pick a random greeting.
+  // Pick a random quote.
   const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function getCommentUsingArrowFunctions() {
```

Comment:
curious: why the "usingArrowFunctions"?

Reply:
I had a getCommentUsingAwait function that did the exact same thing as this one but used the await keyword (because await was more straightforward to read). I changed it to getComment now that I don't have the other function.